HW_7/model.py

```diff
@@ -0,0 +1,28 @@
+from asyncio import constants
+
+
+path = 'logger.py'
+contacts = []
+
+def read_file():
+    global contacts
+    with open(path) as f:
+        contacts = [i.strip().split(';') for i in f.readlines()]
+    return contacts
+
+def get_contacts():
+    global contacts
+    return contacts
+
+def add_contact():
+    global contacts
+    id = input('Введите id: ')
+    name = input('Введите name: ')
+    phone = input('Введите phone: ')
+    comment = input('Введите comment: ')
+    contacts.append(';'.join([id, name, phone, comment]))
+
+def save_file():
+    global contacts
+    with open(path, 'w', encoding='utf_8') as f:
```

Comment:
Ваш файл нельзя будет прочесть у вас кодировка неверно написана.
```suggestion
    with open(path, 'w', encoding='utf-8') as f:
```